orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClient.java

```diff
@@ -95,7 +95,8 @@ public OrchestrationChatResponse chatCompletion(
       throws OrchestrationClientException {
 
     val request = toCompletionPostRequest(prompt, config);
-    return new OrchestrationChatResponse(executeRequest(request));
+    val response = executeRequest(request);
+    return new OrchestrationChatResponse(response);
```

Comment:
(Comment)

For easier troubleshooting when investigating exceptions thrown through line numbers.